dom/manifest/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
BTW, I'm using `.jscsrc` ([jscs](http://jscs.info/)) with the following config:

```
{
    "preset": "google",
    "fileExtensions": [".js", ".jsm"],
    "esnext": true,
    "validateQuoteMarks": true
}
```

I'm flexible tho.